msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/StringHelper.java

```diff
@@ -41,6 +41,39 @@ static private String createSha256Hash(String stringToHash, boolean base64Encode
         return res;
     }
 
+    /**
+     * Creates a SHA-256 hash of the input string and returns it as a lowercase hex string.
+     * This is used for token revocation and other scenarios requiring hex hash representation.
+     *
+     * @param stringToHash The string to hash
+     * @return The SHA-256 hash of the string as a lowercase hex string
+     * @throws MsalClientException If the SHA-256 algorithm is not available
+     */
+    public static String createSha256HashHexString(String stringToHash) {
```

Comment:
nit: nothing outside the package can access StringHelper methods, so no need to be public 

```suggestion
    static String createSha256HashHexString(String stringToHash) {
```

Reply:
Done